### PR TITLE
OP-411 correct deleteAllTherapies() method

### PR DESCRIPTION
**Pull request by dbmalkovsky**
I was writing tests for the Therapy package and noticed that the method `deleteAllTherapies(Integer code)` was not working correctly.

I updated the method body without changing the method signature and my local tests now run cleanly.

This is a separate PR so the change doesn't get lost in the larger JUnit test suite to come.

### CODE REVIEW

**Review by mwithi — CHANGES_REQUESTED** [review]
Maybe we can remove also the copy-and-paste comment on [line 195](https://github.com/informatici/openhospital-core/blob/24c4d4d541b1c85c03328e585eb3650dedebf07b/src/main/java/org/isf/therapy/manager/TherapyManager.java#L195)

**Comment by dbmalkovsky** [comment]
Good catch; made the same type of change as was done originally

File: src/main/java/org/isf/therapy/manager/TherapyManager.java [hunk 1]
```diff
@@ -188,12 +187,10 @@ public TherapyRow newTherapy(TherapyRow thRow) throws OHServiceException {
...
 	public boolean newTherapies(ArrayList<TherapyRow> thRows) throws OHServiceException {
 		if (!thRows.isEmpty()) {
-			DateTime now = new DateTime();
-
 
 			int patID = thRows.get(0).getPatient().getCode();
-			// FIXME FIXME			ioOperations.deleteAllTherapies(patID);
-			smsOp.deleteByModuleModuleID("therapy", String.valueOf(patID));
+			Patient patient = patientManager.getPatientById(patID);
+			ioOperations.deleteAllTherapies(patient);
```
Comment (mwithi) [inline src/main/java/org/isf/therapy/manager/TherapyManager.java:193]: Only the comment was wrong, the operation was correct! 😅

**Review by mwithi — CHANGES_REQUESTED** [review]
Only the comment was wrong...

File: src/main/java/org/isf/therapy/manager/TherapyManager.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/therapy/manager/TherapyManager.java]: OK; let us try it one more time.

**Review by mwithi — APPROVED** [review]

**Comment by mwithi** [comment]
LGTM
```
Results :

Tests run: 1059, Failures: 0, Errors: 0, Skipped: 1
```

**Pull request merged**